assets/css/custom.scss

```
@@ -4250,4 +4250,411 @@ font-size: 1.55rem;
+  }
+  &__track {
+    display: flex;
+    transition: transform 0.65s cubic-bezier(0.33, 0, 0.15, 1);
+    will-change: transform;
+  }
+}
+
+.csh-slide {
+  flex: 0 0 100%;
+  min-width: 100%;
+  // Keep off-screen slides out of the accessibility tree / paint until they
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)